test/components/DateRangePickerInput_spec.jsx

```diff
@@ -117,6 +117,17 @@ describe('DateRangePickerInput', () => {
     });
   });
 
+  describe('props.customCloseIcon is a React Element', () => {
+    it('has custom icon', () => {
+      const wrapper = shallow(
+        <DateRangePickerInput
+          showClearDates
+          customCloseIcon={<span className="custom-close-icon" />}
+        />);
+      expect(wrapper.find('.DateRangePickerInput__calendar-icon .custom-close-icon'));
```

Comment:
This should fail... why would it be under `.DateRangePickerInput__calendar-icon`? Why isn't this failing?